src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+    private User[] users = userService.getUsers();
+
     public boolean login(String email, String password) {
+        for (User user : users) {
+            if (email.equals(user.getEmail()) && password.equals(user.getPassword())) {
+                return true;
+            }
+        }
```

Comment:
What I meant here, is that we should here use our `userService` for user fetching. We do not need again traverse an array of users.